pkg/log/zap/kube_helpers.go

```diff
@@ -41,21 +40,6 @@ type KubeAwareEncoder struct {
 	Verbose bool
 }
 
-// namespacedNameWrapper is a zapcore.ObjectMarshaler for Kubernetes NamespacedName.
-type namespacedNameWrapper struct {
-	types.NamespacedName
-}
-
-func (w namespacedNameWrapper) MarshalLogObject(enc zapcore.ObjectEncoder) error {
-	if w.Namespace != "" {
-		enc.AddString("namespace", w.Namespace)
-	}
-
-	enc.AddString("name", w.Name)
-
-	return nil
-}
```

Comment:
Why are you removing the helpers? 
It does not seem  required  at all.

Reply:
This part of the code never gets called after the k8s patch and the end result becomes the same (MarshalLog() returns lowercase name/namespacename)